project-repo/templates/common/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+title: ""
+labels: ""
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to report an issue and help improve PROJECT_TODO! Please fill out the form below as
+        best as you can so that we can help you.
+
+        Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general
+        support or usage questions, use [Ubuntu Discourse](https://discourse.ubuntu.com/c/desktop/8).
+
+        By submitting an Issue to this repository, you agree to the terms within the
+        [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct).
+
+        Be careful with sensitive information and security vulnerabilities. In order to report bugs that could contain
+        sensitive information, use [Launchpad](https://bugs.launchpad.net/ubuntu/+source/PROJECT_TODO/+filebug) instead.
+        On Ubuntu machines, you can use `ubuntu-bug PROJECT_TODO` to collect relevant information.
+
+
+        FIXME: snap?
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      options:
+        - label: I have searched the existing issues and found none that matched mine
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the issue
+      description: Provide a clear and concise description of what the issue is, including what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce it
+      description: >
+        Detail the steps taken to reproduce this error, what was expected, and whether this issue can be reproduced
+        consistently or if it is intermittent.
+      placeholder: |
+        Where applicable, please include:
+        * Code sample to reproduce the issue
+        * Log files (redact/remove sensitive information)
+        * Application settings (redact/remove sensitive information)
+        * Screenshots
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        Provide details about the environment you experienced the issue in.
+        * Distribution and version
+        * PROJECT_TODO version
+        * Desktop environment
+        * Hardware
+        * etc.
+
+        Ubuntu users can run `ubuntu-bug PROJECT_TODO --save=/tmp/report` and paste the file content bellow.
+
+        FIXME: snap?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Relevant information
+      description: >
+        Please look at our [Troubleshoot guidelines](#) and provide the debug logs for PROJECT_TODO.
+
+        Remember to redact any sensitive information from them.
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Is there any sensitive information in the logs?
+      options:
+        - label: I have redacted any sensitive information from the logs
```

Comment:
what does it do? Can it prevent publishing it? I don’t see information or automation about this.

Reply:
This is supposed to show a checkbox (and the ticking is required) to reinforce the user about the need to redact sensitive information. It does not prevent the user from ticking it and submitting the issue anyway, but it's another layer at least.

Reply:
ok, I agree that’s better than not having it.